cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
       | Decimal-Input    |
       | GroupedCharacter |
       | Number-Input     |
+
+  @accessibility
+  Scenario Outline: Accordion component <story> page closed state
+    When I open "Accordion" component page "<story>"
+    Then "Accordion <story> page" component has no accessibility violations
+    Examples:
+      | story                |
+      | default story        |
+      | with dynamic content |
+
+  @accessibility
+  Scenario: Accordion component primary page opened state
+    Given I open "Accordion" component page "default story"
+    When I expand default accordionRow via click
+    Then "Accordion default page" component has no accessibility violations
+
+  @accessibility
+  Scenario Outline: <component> default story page
+    When I open "Test <component>" component page "default"
+    Then "<component>" component has no accessibility violations
+    Examples:
+      | component        |
+      | Anchornavigation |
+
+  @accessibility
+  Scenario: DuellingPicklist component default story page
+    When I open "DuellingPicklist Test" component page "default"
+    Then "DuellingPicklist" component has no accessibility violations
+
+  @accessibility
+  Scenario Outline: Numeral Date component <story> story page
+    When I open "Numeral Date Test" component page "<story>"
+    Then "Numeral Date <story>" component has no accessibility violations
+    Examples:
+      | story       |
+      | default     |
+      | validations |
+
+  @accessibility
+  Scenario Outline: Button component <story> page
+    When I open "Button" component page "<story>"
+    Then "Button <story> page" component has no accessibility violations
+    Examples:
```

Comment:
Why are all of the Button stories not listed here?

Reply:
we are still working on this for accessibility changes